docs/sources/variables/global-variables.md

```diff
@@ -10,23 +10,32 @@ weight = 200
 
 # Global variables
 
-Grafana has global built-in variables that can be used in expressions in the query editor. This topic lists them in alphabetical order and defines them.
+Grafana has global built-in variables that can be used in expressions in the query editor. This topic lists them in alphabetical order and defines them. These variables are useful in both queries, dashboard links, panel links & data links.
```

Comment:
```suggestion
Grafana has global built-in variables that can be used in expressions in the query editor. This topic lists them in alphabetical order and defines them. These variables are useful in queries, dashboard links, panel links, and data links.
```

Reply:
"Both" means you only have two items in your list. Also, please do not use ampersands as an abbreviation for "and" in prose.